### ISSUE TITLE
Add helper functions

### ISSUE THREAD

**Issue by martinsjek** [sibiraj-s/browser-dtector#8]
Hello, it would be great if you could add some helper functions like: isChrome(), isFirfoex() etc.

**Comment by sibiraj-s** [comment]
Hi. Thanks for creating issue, but where does it stop 🤔 https://github.com/sibiraj-s/browser-dtector/blob/master/lib/constants.ts#L1. Its won't be nice to create helper for all browsers or all systems.

maybe I can expose the constants and you could have your own helper like


```js
browser = detector.parseUserAgent()
isChrome ==== browser.name === KnownBrowsers.chrome
```

Open to discussion on this.

**Comment by martinsjek** [comment]
You could add this for Major browsers only (Chrome, Opera, Firefox, Safari, Edge, IE (because in today's world nobody wants to support IE as for me I use you package to show message that user should update browser version or change to better options)).

![image](https://github.com/sibiraj-s/browser-dtector/assets/36192919/b50d5c82-1e83-452e-9f9b-a1ddc695efc8)

I took this screenshot from PHP Library: https://github.com/hisorange/browser-detect

**Comment by martinsjek** [comment]
Any updates on this?

**Comment by sibiraj-s** [comment]
Hi, Thanks for the ping. Sorry I forgot about this. 

This can be added. instead of functions, is it okay to just have booleans

```js
browser = detector.parseUserAgent()

browser.isChrome //-> true / false
```

**Comment by martinsjek** [comment]
I believe that booleans will work as well ;)

**Comment by martinsjek** [comment]
I see you have released version 4.0.0 - any updates on this?

**Comment by sibiraj-s** [comment]
Apologies for the delay, got stuck in personal work. Its now added in v4.1.0

https://github.com/sibiraj-s/browser-dtector/blob/266ccf28c2ae45211ec5114358bc5a5412208b6c/lib/browser-dtector.ts#L160-L165
